lib/protocol/routes.js

```diff
@@ -700,6 +700,12 @@ const METHOD_MAP = {
       },
     }
   },
+
+  // chromium devtools
+  // https://chromium.googlesource.com/chromium/src/+/master/chrome/test/chromedriver/server/http_handler.cc
+  '/session/:sessionId/goog/cdp/execute': {
+    POST: {}
```

Comment:
Please also add required arguments: https://chromium.googlesource.com/chromium/src/+/master/chrome/test/chromedriver/window_commands.cc#1635

Reply:
and the command name

Reply:
yeah, I was modifying